### ISSUE TITLE
Remove witness_withdraw_pay operation

### ISSUE THREAD

**Issue by bytemaster** [cryptonomex/graphene#142]
Instead pay witnesses into a vesting balance object with a 1 day maturity requirement and then witnesses can use the withdraw_vesting_balance operation to claim the funds.  Fewer operations is less code to maintain.

**Comment by bytemaster** [comment]
This is a low priority hard fork, the current system works as is.   We can consider this change only once all other work has been completed a stable release is ready.

**Comment by vikramrajkumar** [comment]
Stable release is far past protocol freeze. If this is such low priority, then it is wontfix.

**Comment by theoreticalbts** [comment]
Implemented and tested.
